core/arch/interp.c

```diff
@@ -3744,6 +3744,21 @@ build_bb_ilist(dcontext_t *dcontext, build_bb_t *bb)
             break;
         }
 
+#ifdef X86
+        if (my_dcontext != NULL && instr_is_syscall(bb->instr) &&
+            instr_get_opcode(bb->instr) == OP_int &&
+            get_syscall_method() != SYSCALL_METHOD_INT) {
+            instr_t * i2 = INSTR_CREATE_nop(dcontext);
+            SYSLOG_INTERNAL(SYSLOG_WARNING,
+                            "Changing interruption to nop at "PFX"\n", bb->instr_start);
```

Comment:
That was the first POC I could find.
But then, it is clearly not transparent to the application.